seatunnel-formats/seatunnel-format-json/src/test/java/org/apache/seatunnel/format/json/canal/CanalJsonSerDeSchemaTest.java

```diff
@@ -69,6 +73,85 @@ public void testDeserializeNullRow() throws Exception {
         assertEquals(0, collector.list.size());
     }
 
+    @Test
+    public void testDeserializeNoJson() throws Exception {
+        final CanalJsonDeserializationSchema deserializationSchema =
+                createCanalJsonDeserializationSchema(null, null);
+        final SimpleCollector collector = new SimpleCollector();
+        String noJsonMsg = "{]";
+
+        SeaTunnelRuntimeException expected = CommonError.jsonOperationError(FORMAT, noJsonMsg);
+        assertThrows(
+                expected.getClass(),
+                () -> {
+                    deserializationSchema.deserialize(noJsonMsg.getBytes(), collector);
+                },
+                expected.getMessage());
+    }
```

Comment:
```suggestion
        SeaTunnelRuntimeException expected = CommonError.jsonOperationError(FORMAT, noJsonMsg);
        assertThrows(
                expected.getClass(),
                () -> {
                    deserializationSchema.deserialize(noJsonMsg.getBytes(), collector);
                });
    }
```
The message parameter in `assertThrows` only display when assert failed. So I think we shouldn't put `expected.getMessage()` into there.

Reply:
Yes, you are right.